config/resources/domain.json

```diff
@@ -0,0 +1,169 @@
+{
+  "version": "0.1",
+  "prefixes": {
+    "prov": "http://www.w3.org/ns/prov#",
+    "locn": "http://www.w3.org/ns/locn#",
+    "sssom": "https://w3id.org/sssom/",
+    "dct": "http://purl.org/dc/terms/",
+    "ext": "http://mu.semte.ch/vocabularies/ext/",
+    "uitplatform": "https://data.uitwisselingsplatform.be/ns/platform#",
+    "adms": "http://www.w3.org/ns/adms#",
+    "uitdatabank": "https://data.publiq.be/ns/uitdatabank#"
+  },
+  "resources": {
+    "mappings": {
+      "name": "mapping",
+      "class": "sssom:Mapping",
+      "attributes": {
+        "created": {
+          "type": "date",
+          "predicate": "dct:created"
+        },
+        "modified": {
+          "type": "date",
+          "predicate": "dct:modified"
+        },
+        "justification": {
+          "type": "url",
+          "predicate": "sssom:mapping_justification"
+        },
+        "subject-label": {
+          "type": "string",
+          "predicate": "sssom:subject_label"
+        },
+        "object-label": {
+          "type": "string",
+          "predicate": "sssom:object_label"
+        },
+        "score": {
+          "type": "float",
+          "predicate": "sssom:similarity_score"
+        },
+        "subject-uri": {
+          "type": "url",
+          "predicate": "sssom:subject_id"
+        },
+        "object-uri": {
+          "type": "url",
+          "predicate": "sssom:object_id"
+        }
+      },
+      "relationships": {},
+      "features": [
+        "include-uri"
+      ],
+      "new-resource-base": "http://data.publiq.be/mappings/"
+    },
+    "addresses": {
+      "name": "address",
+      "class": "locn:Address",
+      "attributes": {
+        "created": {
+          "type": "date",
+          "predicate": "dct:created"
+        },
+        "modified": {
+          "type": "date",
+          "predicate": "dct:modified"
+        },
+        "full-address": {
+          "type": "language-string",
+          "predicate": "locn:fullAddress"
+        },
+        "postcode": {
+          "type": "string",
+          "predicate": "locn:postCode"
+        },
+        "postname": {
+          "type": "language-string",
+          "predicate": "locn:postName"
+        },
+        "thoroughfare": {
+          "type": "language-string",
+          "predicate": "locn:thoroughfare"
+        },
+        "locator-designator": {
+          "type": "string",
+          "predicate": "locn:locatorDesignator"
+        },
+        "adminunit-l1": {
+          "type": "string",
+          "predicate": "locn:adminUnitL1"
+        },
```

Comment:
I would use more "common" names for these attributes, like `street`, `country`, ...  based on the data